### PR TITLE
Use hash router

### PR DESCRIPTION
**Pull request by polarker**

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@
 import React, { useState, useEffect, useRef, useCallback } from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import { BrowserRouter as Router, Redirect, Route, useLocation } from 'react-router-dom'
+import { HashRouter as Router, Redirect, Route, useLocation } from 'react-router-dom'
```
Comment (tdroxler) [inline src/index.tsx:20]: what's the difference?

Comment (polarker) [reply src/index.tsx:20]: this works better with Github pages: https://create-react-app.dev/docs/deployment/#notes-on-client-side-routing
the url will be like `https://devnet.alephium.org/#/blocks` with a hash character in between

File: src/utils/util.tsx [hunk 2]
```diff
@@ -104,3 +104,7 @@ export function useInterval(callback: () => any, delay: number, shouldPause = fa
     }
   }, [delay, shouldPause]);
 }
+
+export function smartHash(hash: string) {
+  return hash.substring(0, 8) + '...' + hash.substring(hash.length - 8)
```
Comment (tdroxler) [inline src/utils/util.tsx:102]: maybe we should have something like 
```
if (hash.length < 10)
   return hash
else
  return hash.substring(0, 8) + '...' + hash.substring(hash.length - 8)
```
we currently don't use this function on short hash, but nothing prevent us to do it in the future

File: src/components/Amount.tsx [hunk 3]
```diff
@@ -22,6 +22,7 @@ interface AmountProps {
 }
 
 const Amount: FC<AmountProps> = ({ value, className }) => {
+  console.log('value' + value?.toString())
```
Comment (tdroxler) [inline src/components/Amount.tsx:25]: is it a debug leftover or you really want it?

**Comment by tdroxler** [comment]
great to start adding tests :+1:

**Review by tdroxler — APPROVED** [review]
minor comments that you can tackle or not, otherwise lgtm

File: src/components/Amount.tsx (hunk 3 above)
Comment (polarker) [reply src/components/Amount.tsx:25]: Nice catch!

**Pull request merged**